packages/shared/src/sentry.ts

```diff
@@ -0,0 +1,114 @@
+import * as Sentry from "@sentry/node";
+const { nodeProfilingIntegration } = require("@sentry/profiling-node");
+import { AuthInfo } from "./graphql/fetchAuthInfoFromGraphQL";
+import { getDeviceId } from "./getDeviceId";
+import { randomUUID } from "crypto";
+import { logger } from "./logger";
+import { formatTags, Tags } from "./formatTags";
+
+const SENTRY_DSN =
+  "https://5c145b72bb502832982243d6584f163d@o437061.ingest.us.sentry.io/4507534763819008"; // write-only permissions
+
+class SentryAPI {
+  private deviceId: string;
+  private sessionId: string;
+  private authInfo?: AuthInfo;
+  private initialized: boolean = false;
+
+  constructor() {
+    this.deviceId = getDeviceId();
+    this.sessionId = randomUUID();
+  }
+
+  initialize(app: string, version: string | undefined) {
+    if (this.initialized) {
+      console.warn(`Logger already initialized.`);
+    }
+
+    Sentry.init({
+      dsn: SENTRY_DSN,
+      integrations: [nodeProfilingIntegration()],
+    });
+
+    Sentry.setTags({
+      app,
+      version,
+    });
+
+    this.initialized = true;
+  }
+
+  identify(authInfo: AuthInfo) {
+    this.authInfo = authInfo;
+  }
+
+  async close() {
```

Comment:
we should add a call to this to `exitTasks` here:
https://github.com/replayio/replay-cli/blob/207cf051bcbc008da3290b23ae2a466d44f56f45/packages/shared/src/process/exitTask.ts#L7